recipes/datefinder/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "datefinder" %}
+{% set version = "0.7.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/akoumjian/datefinder/archive/v0.7.1.tar.gz"
```

Comment:
`url: "https://github.com/akoumjian/datefinder/archive/v{{ version}}.tar.gz"`

Also in the future if https://github.com/akoumjian/datefinder/pull/130 gets merged, you can use the pypi release without having to manually package the license file.

Reply:
Done -- ACK on the LICENSE